tracing/tests/macros.rs

```diff
@@ -533,7 +533,7 @@ fn error_root() {
 
 #[test]
 fn event_with_parent() {
-    let p = span!(Level::TRACE, "im_a_parent!");
+    let _p = span!(Level::TRACE, "im_a_parent!");
```

Comment:
This shouldn't be necessary any more. You probably added this for the warning that was emitted due to the `parent:` bug. May also need to be undone elsewhere.

Reply:
rustfix did this, will make sure the parent bug is actually resolved